phoenix-core/src/it/java/org/apache/phoenix/end2end/SortMergeJoinMoreIT.java

```diff
@@ -39,10 +39,10 @@
 
 import com.google.common.collect.Maps;
 
-public class SortMergeJoinMoreIT extends BaseHBaseManagedTimeIT {
+public class SortMergeJoinMoreIT extends BaseHBaseManagedTimeTableReuseIT {
```

Comment:
It doesn't look like all test methods were converted to use generateRandomString() for table names.